samples/dynamodb/lambda/DynamoDB_1/ClientUI/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using System;
+using NServiceBus;
+
+class Program
+{
+  public static async Task Main(string[] args)
+  {
+    var endpointConfiguration = new EndpointConfiguration("Samples.DynamoDB.Lambda.ClientUI");
+    endpointConfiguration.SendFailedMessagesTo("Samples-DynamoDB-Lambda-Error");
```

Comment:
configured this endpoint to use the same error queue

Reply:
also disabled the installers for the client and added the necessary resources to the cloud formation template instead.